arduino/smartcar/smartcar.ino

```diff
@@ -42,23 +46,34 @@ void setup() {
 #ifdef __SMCE__
 Camera.begin(QQVGA, RGB888, 15);
   frameBuffer.resize(Camera.width() * Camera.height() * Camera.bytesPerPixel());
-  //mqtt.begin("3.138.188.190", 1883, WiFi);
+  mqtt.begin("3.138.188.190", 1883, WiFi);
   //mqtt.begin("aerostun.dev", 1883, WiFi);
-  mqtt.begin(WiFi); // Will connect to localhost
+  //mqtt.begin(WiFi); // Will connect to localhost
 #else
   mqtt.begin(net);
 #endif
   if (mqtt.connect("arduino", "public", "public")) {
     mqtt.subscribe("/smartcar/control/#", 1);
     mqtt.onMessage([](String topic, String message) {
```

Comment:
Similar to @JihanP, the ino file doesnt compile for me either when testing. I believe it is due to a bug relating to this line. This bug has been fixed, and the fix is on the master branch - so merging should resolve this issue.